api4/user.go

```diff
@@ -2283,6 +2283,10 @@ func verifyUserEmail(c *Context, w http.ResponseWriter, r *http.Request) {
 	auditRec.Success()
 	c.LogAudit("Email Verified")
 
+	c.App.Srv().GetTelemetryService().SendTelemetry("email_verified", map[string]any{
```

Comment:
I just checked and there's an event named `verify_email` that is sent from the front-end. Do we need both?

Properties should also include `user_actual_id`. This will allow for correlating which users have verified their emails.